src/helpers/utils.ts

```diff
@@ -24,6 +24,19 @@ export const getConfig = async (configPath: string) => {
   return JSON.parse(configFile)
 }
 
+/**
+ * Setting the API Key and site as environment variables is mandatory to call the metrics
+ * logger. Call this function if you read their values from somewhere else (e.g. `datadog-ci.json`).
```

Comment:
According to `datadog-metrics` documentation it is not mandatory to pass the API key through an environment variable, it is possible to set it in the `.init` function of datadog metrics.
cf. https://github.com/dbader/node-datadog-metrics#initialization

Reading the API key from a file, then setting it an environment variable to later re-read it from a dependency seems quite indirect, could we init `datadog-metrics` with the API key directly instead?